packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,181 @@
+  flow,
+  colorMode = 'light',
+  visualAsBackground = false,
+  children,
+  ...rest
+}: BentoItemProps) => {
+  const bentoItemClassArray = [styles.Bento__Item]
+  bentoItemClassArray.push(
+    ...returnClassBasedOnResponsiveMap('column-span', columnSpan),
+    ...returnClassBasedOnResponsiveMap('row-span', rowSpan),
+    ...returnClassBasedOnResponsiveMap('column-start', columnStart),
+    ...returnClassBasedOnResponsiveMap('row-start', rowStart),
+    ...returnClassBasedOnResponsiveMap('flow', flow),
+  )
+
+  return (
+    <div
+      data-color-mode={colorMode}
+      className={clsx(
+        ...bentoItemClassArray,
+        visualAsBackground && styles[`Bento__Item--visual-as-background`],
+        className,
+      )}
+      {...rest}
+    >
+      {children}
+    </div>
+  )
+}
+
+type BentoContentProps = {
+  leadingVisual?: ReactElement | Icon
+  padding?: 'condensed' | 'normal' | 'spacious'
+  verticalAlign?: ResponsiveAlign
+  horizontalAlign?: ResponsiveAlign
+  fixedBottomLink?: boolean
+} & React.HTMLAttributes<HTMLDivElement> &
+  BaseProps<HTMLDivElement>
+
+const Content = ({
+  children,
+  leadingVisual: LeadingVisual,
+  padding = 'normal',
+  verticalAlign = 'start',
+  horizontalAlign = 'start',
+  fixedBottomLink = false,
+  className,
+  ...rest
+}: BentoContentProps) => {
+  const bentoContentClassArray = [styles.Bento__Content]
+  bentoContentClassArray.push(
+    ...returnClassBasedOnResponsiveMap('verticalAlign', verticalAlign),
+    ...returnClassBasedOnResponsiveMap('horizontalAlign', horizontalAlign),
+  )
+  const HeadingChild = React.Children.toArray(children).find(
+    child => React.isValidElement(child) && child.type === Heading,
+  )
+  const TextChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Text)
+  const LinkChild = React.Children.toArray(children).find(child => React.isValidElement(child) && child.type === Link)
+  return (
+    <div className={clsx(styles[`Bento-padding--${padding}`], ...bentoContentClassArray, className)} {...rest}>
+      {React.isValidElement(LeadingVisual) &&
+        React.cloneElement(LeadingVisual as React.ReactElement<IconProps>, {
+          size: LeadingVisual['size'] || 32,
+        })}
+      {React.isValidElement(HeadingChild) &&
+        React.cloneElement(HeadingChild as React.ReactElement<HeadingProps>, {
+          as: HeadingChild.props.as || 'h3',
+          size: HeadingChild.props.size || '3',
+        })}
+      {React.isValidElement(TextChild) &&
+        React.cloneElement(TextChild as React.ReactElement<TextProps>, {
+          variant: 'muted',
+          as: 'p',
+        })}
+      {React.isValidElement(LinkChild) &&
+        React.cloneElement(LinkChild as React.ReactElement<LinkProps>, {
+          className: clsx(
+            LinkChild.props.className,
+            styles['Bento__call-to-action'],
+            fixedBottomLink && styles['Bento__call-to-action--fixed'],
+          ),
+        })}
```

Comment:
Should this be the default? I would imagine this should be configured by the user to some degree. If we have this as default then users will need to manage how this will appear on different backgrounds and how they want to manage color modes.

Reply:
This has been updated in the following SHAs:
https://github.com/primer/brand/pull/390/commits/0a88f2328c740141414e178c219f6821d67de906
https://github.com/primer/brand/pull/390/commits/fedfe4234b082cc90d999ab00972e39402df8db3